harness/determined/exec/gc_checkpoints.py

```diff
@@ -64,6 +76,12 @@ def main(argv: List[str]) -> None:
         default=os.getenv("DET_DELETE", []),
         help="Checkpoints to delete (JSON-formatted file)",
     )
+    parser.add_argument(
+        "--delete-tensorboards",
+        action="store_true",
+        default=False,
```

Comment:
maybe accept the environment variable to be consistent?  Or delete all the env variables that we haven't used in like forever...

Either way.